lib/overcommit/hook/post_commit/git_guilt.rb

```diff
@@ -0,0 +1,9 @@
+module Overcommit::Hook::PostCommit
+  # Calculates the change in blame since the last revision.
+  class GitGuilt < Base
+    def run
+      system(executable, 'HEAD~', 'HEAD')
```

Comment:
It sounds like `git guilt` only formats its output with newlines and color escape sequences if its writing to a TTY. Since in this case we're intercepting the output, it doesn't output color codes.

It's unfortunate it doesn't seem to have a `--color` flag to force color output, which is the standard solution to this sort of problem.

I'd rather not make an exception to using `execute` just for this feature. It should be easy enough to parse the output inline and manually output colors on its behalf. I realize Overcommit doesn't provide nice helpers for this, so for now you can just output the escape codes manually and we'll look into providing some helpers later.